group_vars/all.yaml

```diff
@@ -9,8 +9,8 @@ allow_delete: false
 
 # Important: The URL's for engine releases contain %7B characters for brackets. It seems that wget does not always like these, so replace with {}
 # Note that these characters get replaced with '.' characters for the engine_dir
-engine_github_path: https://github.com/beyond-all-reason/spring/releases/download/2025.01.6/spring_bar_.rel2501.2025.01.6_linux-64-minimal-portable.7z
-engine_dir: spring_bar_.rel2501.2025.01.6_linux-64-minimal-portable
+engine_github_path: https://github.com/beyond-all-reason/spring/releases/download/2025.03.9/recoil_2025.03.9_amd64-linux.7z
+engine_dir: recoil_2025.03.9
```

Comment:
This is incorrect, you repeat https://github.com/beyond-all-reason/ansible-spads-setup/issues/54. If you tested this with `--check --diff`, you would see that it shows invalid diff, the correct path is `recoil_2025.03.9_amd64-linux`.

Reply:
I've pushed commit to fix this, I need this change in.